### DISCUSSION TITLE
Alt Switching

### DISCUSSION THREAD

**Discussion by Fluffy728** [ScratchAddons/ScratchAddons#6160] [Ideas] [answered]
So, there will be an option to enable alt switching, so in the account dropdown, there will be a button saying Switch to alt. Of course, you will have to sign in to it in the first place when you first linked it.

**Comment by CST1229** [comment, marked as answer]
See #1074

**Reply by Fluffy728** [reply]
Duplicate alert, closing

**Comment by Fluffy728** [comment]
Duplicate
